README.md

```diff
@@ -31,10 +32,115 @@ sudo kubeadm join 10.10.1.1:6443 --token btytkp.7nh8pawcdsi23g4x \
 	--discovery-token-ca-cert-hash sha256:9d1802d5451e559b5c076db6901865b164bd201ed46ce38c1cba03e89618e027 \
   --cri-socket unix:///var/run/cri-dockerd.sock
 ```
+**Note:** add config from *master* node to *worker* node.
+
+On *master* node, copy contents of .kube/config `cat ~/.kube/config`
+
+On *worker* node, create ~/.kube/config and paste content 
+```
+mkdir ~/.kube
+sudo vim ~/.kube/config
+```
+To check cluster run `kubectl get nodes`
 
 6. run `echo 'source <(kubectl completion bash)' >>~/.bashrc && source ~/.bashrc`
 
 ## Install Knative
 ```
 ./400-kn-install.sh
 ```
+
+## Setup Flame (on Worker node)
+1. Run `./500-flame-setup.sh`
+2. Add urls to /etc/hosts using *master* node ip address.
+Example
+```
+128.110.218.153	apiserver.flame.test
+128.110.218.153	notifier.flame.test
+128.110.218.153	mlflow.flame.test
+128.110.218.153	controller.flame.test
+```
+3. On *master* node install haproxy `sudo apt-get install haproxy`
+4. Setup load balancing with haproxy ` sudo vim /etc/haproxy/haproxy.cfg`
+**Note:** change srv1 and srv2 using ingress-nginx-controller ports from `kubectl get svc -A` 
+
+Example code snippet (add to end of haproxy.cfg):
+```
+listen l1
```

Comment:
What do "l1" and "l2" represent for?

Reply:
l1 and l2 are for redirecting :443 and :80 ports to nginx-ingress controller.